quickcomm/forms.py

```diff
@@ -40,19 +38,18 @@ class CreateMarkdownForm(forms.Form):
     """A form for creating a markdown post."""
 
     title = forms.CharField(max_length=100)
-    source = forms.URLField(validators=[URLValidator])
-    origin = forms.URLField(validators=[URLValidator])
     description = forms.CharField(max_length=1000)
     content = MartorFormField()
     categories = forms.CharField(max_length=1000)
     visibility = forms.ChoiceField(choices=Post.PostVisibility.choices)
     unlisted = forms.BooleanField(required=False)
+    friends_only=forms.BooleanField(required=False)
```

Comment:
why is this line here and in image post but not for text post?

as far as I can tell it's extraneous and no longer does anything